recipes/minizip/all/conanfile.py

```diff
@@ -0,0 +1,138 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class MinizipConan(ConanFile):
+    name = "minizip"
+    description = "minizip is a zip manipulation library written in C "
+    "that is supported on Windows, macOS, and Linux."
+    topics = ("conan", "minizip", "compression")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/nmoinvaz/minizip"
+    license = "Zlib"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake_find_package"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "compat": [True, False],
+        "zlib": [True, False],
+        "bzip2": [True, False],
+        "zstd": [True, False],
+        "lzma": [True, False],
```

Comment:
This makes it clear to consumers that an option will add a requirement.
```suggestion
        "with_zlib": [True, False],
        "with_bzip2": [True, False],
        "with_zstd": [True, False],
        "lwith_zma": [True, False],
```